src/App.js

```diff
@@ -27,6 +41,7 @@ function App() {
         </Navbar.Collapse>
         </Navbar>
       </header>
+      <Emulator panelData={data} />
```

Comment:
Is the idea that `header` always stays, but the `Emulator` element would swap out based on whether you're on splash screen, curriculum summary, etc?

Reply:
I think we would have top-level components that contain everything (header and footer included). Clicking on the tabs of the NavBar would cause a different top-level component to be rendered (see this index.js from [my weather project](https://github.com/sammytran277/weather-moods/blob/master/src/index.js)). This might be better than just swapping out the middle and leaving the header and footer because I currently don't have plans to add a footer to the emulator wireframe because I would prefer the page to not be scrollable. The actual panels (instructions and/or tabs) could have scrollbars though.